pkgdown/_pkgdown.yml

```diff
@@ -14,6 +14,22 @@ authors:
 
 template:
   bootstrap: 4
+  bslib:
```

Comment:
I'll remove this later, these are the variables from @apreshill's initial comment, plus "component-active-bg" as secondary is quite dark here.